clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -100,17 +100,24 @@ public HelixParticipationService(
       String zkAddress,
       String clusterName,
       int port,
+      String hostname,
       CompletableFuture<SafeHelixManager> managerFuture) {
     this.ingestionService = storeIngestionService;
     this.storageService = storageService;
     this.clusterName = clusterName;
     // The format of instance name must be "$host_$port", otherwise Helix can not get these information correctly.
-    this.participantName = Utils.getHelixNodeIdentifier(port);
+    this.participantName = Utils.getHelixNodeIdentifier(hostname, port);
     this.zkAddress = zkAddress;
     this.veniceConfigLoader = veniceConfigLoader;
     this.helixReadOnlyStoreRepository = helixReadOnlyStoreRepository;
     this.metricsRepository = metricsRepository;
-    this.instance = new Instance(participantName, Utils.getHostName(), port);
+    this.instance = new Instance(participantName, hostname, port);
+    LOGGER.info(
+        "Instance name: {}, hostname: {}, port: {}, url {}",
```

Comment:
I don't mind adding the extra logging, but these info are a bit redundant, aren't they? the `participantName` is just the concatenation of `hostname` and `port`, right?

Reply:
it  is a left over, removed